argh/src/lib.rs

```diff
@@ -972,7 +974,13 @@ impl<'a> ParseStructOptions<'a> {
             .arg_to_slot
             .iter()
             .find_map(|&(name, pos)| if name == arg { Some(pos) } else { None })
-            .ok_or_else(|| unrecognized_argument(arg))?;
+            .ok_or_else(|| {
+                unrecognized_argument(
+                    arg,
+                    self.arg_to_slot,
+                    &["--help".to_owned(), "help".to_owned()],
```

Comment:
After merging #106, this can be updated to include all the help triggers.

Reply:
Fixed in https://github.com/google/argh/pull/180/commits/364f7a85dacf56c70f0977e6999cea4980f7aec6